plugins/flatpak/gs-flatpak.c

```diff
@@ -3972,6 +4033,13 @@ gs_flatpak_update_app (GsFlatpak *self,
 		}
 	}
 
+	if (gs_utils_app_is_auto_updating (app) &&
+	    !gs_flatpak_has_space_to_install_or_update (self, app, list, TRUE)) {
+		g_debug ("Skipping automatic update for: %s as prone to failure due to disk-space",
```

Comment:
I am not much convinced with "no space left on disk" error message.
We are logging the message(which is not user-visible) of a situation that we are trying to avoid.

Reply:
If I understand correctly, this message is printed when the amount of free space left on the disk is less than the minimum we estimated for the given app, so it’s not really a case of there being `no space left on disk`. However, I think `prone to failure due to disk-space` could be clearer.

How about `Skipping automatic update for ‘%s’, as it will likely run out of disk space`?

Reply:
Okay. I just wanted to make it consistent with other error messages, but you're right it's not like we have 0 bytes of space left.

Reply:
BTW, I'd still prefer something more compact like `Skipping automatic update of '%s': not enough space on disk`. Where "enough" means we decided it wasn't sufficient for a safe update...
In any case, I am also fine with Philip's proposal even if it seems more verbose to me.